### DISCUSSION TITLE
"Buy Event" and "Open Event" Chests

### DISCUSSION THREAD

**Discussion by djravine** [djravine/idlecombos#16] [Ideas]
## Event Chests - IN PROGRESS - REQUIRES TESTING
I have started work on this feature.
Will need to check a few things first to make sure it will work.
**THIS WILL BE DISABLED IN RELEASES UNTIL I CAN TEST IT**

**Comment by djravine** [comment]
New Issue https://github.com/djravine/idlecombos/issues/18

**Comment by djravine** [comment]
Released

**Comment by raxele123** [comment]
If you click cancel or x in any buy or open chests all hell breaks loose. You'll know what I mean when you try it. Code sure is weird. The heck made that happen.

**Reply by djravine** [reply]
Whoops, My bad. I forgot to put in a few `return` calls.
Still getting to know all the nuisances of AHK

Fixed in release [v3.40](https://github.com/djravine/idlecombos/releases/tag/v3.40)
